File: UI/wwwroot/Questionnaires/FrontModel/Resultados_Tests.js
//@ts-check
import { EntityClass } from "../../WDevCore/WModules/EntityClass.js";
//@ts-ignore
import { Resultados_Pregunta_Tests } from "./Resultados_Pregunta_Tests.js";
import { Tests } from './Tests.js';
class Resultados_Tests extends EntityClass {
    /** @param {Partial<Resultados_Tests>} [props] */
   constructor(props) {
       super(props, 'EntityQuestionnaires');
       for (const prop in props) {
           this[prop] = props[prop];
       }
   }
   /**@type {Number}*/ Id_Perfil;
   /**@type {Date}*/ Fecha;
   /**@type {String}*/ Seccion;
   /**@type {String}*/ Valor;
   /**@type {String}*/ Categoria_valor;
   /**@type {String}*/ Tipo;
   /**@type {Tests} ManyToOne*/ Tests;
   /**@type {Array<Resultados_Pregunta_Tests>} OneToMany*/ Resultados_Pregunta_Tests;
}
export { Resultados_Tests };

File: BusinessLogic/Questionnaires/Mapping/Model/Resultados_Tests.cs
using CAPA_DATOS;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
namespace DataBaseModel {
   public class Resultados_Tests : EntityClass {
       [PrimaryKey(Identity = true)]
       public int? Id_Resultado { get; set; }
       public int? Id_Perfil { get; set; }
       public int? Id_test { get; set; }
       public DateTime? Fecha { get; set; }
       public string? Seccion { get; set; }
       public DateTime? Created_at { get; set; }
       public DateTime? Updated_at { get; set; }
       public string? Valor { get; set; }
       public string? Categoria_valor { get; set; }
       public string? Tipo { get; set; }
       [ManyToOne(TableName = "Tests", KeyColumn = "Id_test", ForeignKeyColumn = "Id_test")]
       public Tests? Tests { get; set; }
       [OneToMany(TableName = "Resultados_Pregunta_Tests", KeyColumn = "Id_Resultado", ForeignKeyColumn = "Id_Resultado")]
       public List<Resultados_Pregunta_Tests>? Resultados_Pregunta_Tests { get; set; }
   }
}
